src/EnergyPlus/Plant/LoopSide.cc

```diff
@@ -1834,6 +1841,13 @@ namespace DataPlant {
                         SimulateSinglePump(state, PumpLocation, FlowRequest);
                     }
                     break;
+                case OpScheme::WetBulbRB:
+                case OpScheme::DryBulbRB:
+                case OpScheme::DewPointRB:
+                case OpScheme::RelHumRB:
+                case OpScheme::DryBulbTDB:
+                case OpScheme::WetBulbTDB:
+                case OpScheme::DewPointTDB:
```

Comment:
Enable all outdoor air-based plant equipment operation objects to follow the same approach than the load ranges based one